File: Frontend/src/app/models/Actor.ts
export interface IActorDTO {
  id: number;
  nombre: string;
  fechaNacimiento: Date;
  foto: string;
  biografia: string;
}

export interface IActorCreacionDTO {
  nombre: string;
  fechaNacimiento: Date;
  foto: File;
  biografia: string;
}

export interface IActorPeliculaDTO {
  id: number;
  nombre: string;
  personaje: string;
  foto: string;
}

File: Backend/Backend/Models/Actor.cs
﻿using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Backend.Models
{
	public class Actor
	{
		public int Id { get; set; }
		[Required]
		[StringLength(maximumLength: 200)]
		public string Nombre { get; set; }
		public string Biografia { get; set; }
		public DateTime FechaNacimiento { get; set; }
		public string Foto { get; set; }
		// Si yo tengo un actor, me permitirá traer todas las películas
		// en las que ha actuado.
		public List<PeliculasActores> PeliculasActores { get; set; }
	}
}
